monitoring/definitions/postgres.go

```diff
@@ -0,0 +1,119 @@
+package definitions
+
+import (
+	"time"
+
+	"github.com/sourcegraph/sourcegraph/monitoring/monitoring"
+)
+
+const (
+	dbCodeIntel   = "codeintel-db"
+	dbSourcegraph = "pgsql"
+)
+
+func Postgres() *monitoring.Container {
+	return &monitoring.Container{
+		Name:        "postgres",
+		Title:       "Postgres",
+		Description: "Metrics from postgres_exporter.",
+		Groups: []monitoring.Group{
+			{
+				Title: "Default postgres dashboard",
+				Rows: []monitoring.Row{
+					{
+						monitoring.Observable{
+							Name:              "connections",
+							Description:       "connections",
+							Owner:             monitoring.ObservableOwnerCloud,
+							Query:             "sum by (datname) (pg_stat_activity_count{datname!~\"template.*|postgres|cloudsqladmin\"})",
+							DataMayNotExist:   true,
+							DataMayNotBeNaN:   false,
+							Warning:           monitoring.Alert().LessOrEqual(5).For(5 * time.Minute),
+							PossibleSolutions: "none",
+						},
+						monitoring.Observable{
+							Name:              "transactions",
+							Description:       "transaction_durations",
```

Comment:
```suggestion
							Description:       "transaction durations",
```

![image](https://user-images.githubusercontent.com/23356519/101453229-fc2ba480-3969-11eb-8f72-172b139916aa.png)